### DISCUSSION TITLE
Easy ISO install

### DISCUSSION THREAD

**Discussion by Hierosme** [trunkmaster/nextspace#379] [Q&A]
Hello,

I try to make a Video for present NextSpace,  unfortunally it's a bit long  to download CentOS, build, then test under a presention video. (I have make it,  but from my point that is bad idea to publish a video it look how that is long to install)

Have you plan to create a ISO installation for simplify demonstration (something like a LiveCD)?

Regards

**Comment by trunkmaster** [comment]
@Hierosme, I have no plans to create ISO at this point. I guess it's possible to present NextSpace in action without making ISO images.

**Comment by zeroheure** [comment]
@Hierosme  Have a look to armm77 iso project
https://github.com/armm77/nextspace-iso-installer
